brewblox_devcon_spark/api/object_api.py

```diff
@@ -284,34 +313,34 @@ def __init__(self, app: web.Application):
 
 
 @routes.get('/objects')
-async def object_all(request: web.Request) -> web.Response:
+async def all_objects(request: web.Request) -> web.Response:
     """
     ---
-    summary: List all objects
+    summary: List all active objects
     tags:
     - Spark
     - Objects
-    operationId: controller.spark.objects.all
+    operationId: controller.spark.objects.active
     produces:
     - application/json
     """
     return web.json_response(
-        await ObjectApi(request.app).all()
+        await ObjectApi(request.app).list_active()
     )
 
 
-@routes.get('/data')
-async def object_all_data(request: web.Request) -> web.Response:
+@routes.get('/saved')
```

Comment:
endpoints swapped for saved objects and active objects. use saved_objects and objects

Reply:
fixed